Server/controllres/animeGiphy.js

```diff
@@ -0,0 +1,10 @@
+const dotenv = require('dotenv').config();
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
something weird in your code, you're better off leaving a comment explaining what you've done here and why

Reply:
This is from node-fetch npm package. Fetch wasn't working with just require so we read the documentation and it said to add this line to define fetch.